Assets/Plugins/Editor/Uplift/UpliftManager.cs

```diff
@@ -100,7 +100,9 @@ public void InstallDependencies(InstallStrategy strategy = InstallStrategy.UPDAT
 		public DependencyState[] GetDependenciesState()
 		{
 			Upbring upbring = Upbring.Instance();
-			PackageRepo[] targets = GetTargets(GetDependencySolver(), InstallStrategy.UPDATE_LOCKFILE, false);
+			PackageRepo[] targets = GetTargets(GetDependencySolver(),
+											   InstallStrategy.UPDATE_LOCKFILE,
```

Comment:
weird indentation

Reply:
Yes, I saw that. I don't know why it looks like this here.
For readability, I specified one parameter per line, as followed :
```
PackageRepo[] targets = GetTargets(GetDependencySolver(),
			           InstallStrategy.UPDATE_LOCKFILE,
			           updateLockfile: false);
```
If you prefer it on a single line, I can change that quickly.

Reply:
Multiline is fine, but there's something weird with the tabulation vs space I guess.

Reply:
I'm going to check that :)